opencart2.0.x-2.2.x/upload/catalog/controller/payment/xenditcc.php

```diff
@@ -145,6 +151,14 @@ public function process_3ds() {
                     'store_name' => $store_name
                 )
             );
+            
+            if ($charge['status'] !== 'CAPTURED') {
```

Comment:
the original logic lives inside `process_order` might want to merge it there instead